### ISSUE TITLE
how to set the learning rate

### ISSUE THREAD

**Issue by Anikily** [Engineering-Course/LIP_JPPNet#50]
when i run train_JPPNet-s2.py.i found that the loss don't convergence,so i want to know how to set learning rate when train the joint model or is there some principle?
